rrfs-test/testinput/rrfs_mpasjedi_2024052700_Ens3Dvar.yaml

```diff
@@ -125,7 +125,7 @@ cost function:
            defer to post: true
 
 test:
-  reference filename: data/ref/rrfs-mpasjedi-ens3dvar.ref
+  reference filename: testoutput/rrfs-mpasjedi-ens3dvar.ref
```

Comment:
@SamuelDegelia-NOAA To confirm, if we don't link testoutput/ and don't do "test",  jedi_var.x will still run without an error, right?

Reply:
If we do not link testoutput in, JEDI will complete (outputting all files) but it will throw an error at the end for the reference mismatch error. 

I would like to keep these yaml files in `rrfs-test/testinput` as inputs designed for the ctests. If you are worried about using these yamls for separate experiments, then we could either 1) add a line in `setup_experiment.sh` to copy `testoutput` into the run directory, or 2) use different yaml files.

Reply:
> If we do not link testoutput in, JEDI will complete (e.g., outputting all analysis or bumploc files) but it will throw an error at the end for the reference mismatch error.
> 

If it throws an error even if we don't do ctest, then we need to address this problem in this PR: 

1. modify `link_mpasjedi_expr.sh` to link `testoutput` instead of 'ref` to under data/
Or
2. put the ref files still under `fix/expr_data/mpas_2024052700/ref` but without links. We can track them directly. 
Thanks!

Reply:
I prefer solution #&#x2060;1 to keep all the test reference data in one location and to be consistent with other JCSDA repos. I will update the PR to do this.

Reply:
Works for me. Just make sure we can run standalone mpasjedi tests without an error. Thanks

Reply:
Done. Just updated `link_mpasjedi_expr.sh` to link testouput into `expr/mpas_2024052700`. I ran `run_jedi.sh` in there and it passes without error.